pages/firebase-outputs.js

```diff
@@ -0,0 +1,82 @@
+import { useEffect, useState } from 'react';
+import Link from 'next/link';
+
+import {
+  getListOfAllArticlesByAlphabet,
+  getListOfAllArticlesByDate,
+  getListOfLastArticles,
+} from '../firebase/api/articles';
+
+const FirebaseFirst = () => {
+  const [articleList, setArticleList] = useState([]);
+  const [latestArticles, setLatestArticles] = useState([]);
+  const [alphabeticalListOfArticles, setAlphabeticalListOfArticles] = useState(
+    [],
+  );
+
+  useEffect(() => {
+    const listOfArticles = getListOfAllArticlesByDate();
+    listOfArticles.then((value) => setArticleList(value));
+
+    const listOfLastestArticles = getListOfLastArticles();
+    listOfLastestArticles.then((value) => setLatestArticles(value));
```

Comment:
Sorting is definitely better done on the firebase side, but this (lastArticles) is simple `articleList.slice(3)`.

You could save API calls with the identical result

Reply:
I know, but I only want to know, if I can write the code which works. I probably won't use the function where I list all articles ordered by date.